coalib/io/FileFactory.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
```

Comment:
I'm not sure if we should just call it `File` :sweat_smile: But let's shift that discussion after merge :)